ecommerce/static/js/cart.js

```diff
@@ -0,0 +1,43 @@
+var updateBtns = document.getElementsByClassName('update-cart')
+
+for(var i = 0; i < updateBtns.length; i++){
+    updateBtns[i].addEventListener('click', function(){
+        var productId = this.dataset.product
+        var action = this.dataset.action
+        console.log('productId:', productId, 'action:', action)
+
+        console.log('USER:', user)
+        if (user === 'AnonymousUser'){
+            console.log('Nie zalogowany')
+        }
+        else
+        {
+            updateUserOrder(productId, action)
+        }
+    })
+}
+
+
+function updateUserOrder(productId, action){
+    console.log('Zalogowany urzytkownik!')
+
+    var url = '/update_item/'
+    console.log(action)
+    fetch(url, {
+        method: 'POST',
+        headers:{
+            'Content-Type':'application/json',
+            'X-CSRFToken': csrftoken,
+        },
+        body:JSON.stringify({'productId': productId, 'action': action})
+    })
+
+    .then((response) =>{
+        return response.json()
+    })
+
+    .then((data) =>{
+        console.log('data:', data)
+        location.reload()
```

Comment:
o tym już gadaliśmy. Tutaj zgodnie ze sztuką powinno nastąpić podstawienie na stronie w miejsce starej wartości, nowa, zupdatowana już wartość zamiast odświeżać całą stronę. Bo używanie promis (czyli ten fetch then) jest właśnie po to, żeby nie trzeba było odświeżać całej strony.

A tamten jsonowy widok we views.py - update_item - powinien zwracać właśnie nową wartość, którą wkleiłbyś na stronie w odpowiednim miejsciu.